portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,104 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+//HELPER_FUNCTION
+/**Create a typewriter from a given div */
```

Comment:
PTAL at JS style guide on comments https://google.github.io/styleguide/htmlcssguide.html.

Similar to HTML comments, want a space after the // and /*

Here and below.